Edit Button [PAR-51]

Gatlinn opened the pull request:
Creating a new pull request because I had to revert my last PR

Gatlinn committed:
Edit Button [PAR-51]

I fixed the issue with the book table needing to refresh the page

Gatlinn committed:
Merge branch 'master' into library-page

Hashiphoto commented:
Did the new BookEditButton make it in? It doesn't appear to be in this PR. 

The revert blew away all your changes, so I think you have to add them all back. You can take a look at https://github.com/ParaLibrary/paralibrary-frontend/commit/992aa34df76f33b6cacaa7f894bf399e80cf91fe to see what was merged originally.

Gatlinn committed:
Edit Button [PAR-51]

I redid the files but everything was deleted and I quickly pulled my changes from my old commit so if something is out of sorts let me know. It is working great for me.

Hashiphoto reviewed (approved):
Looks good!
